4-fleetscope/modules/env_baseline/acm.tf

```
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  fleet_membership_re = "//gkehub.googleapis.com/(.*)$"
+}
+
+resource "google_project_service" "acm" {
+  project = var.project_id
+  service = "anthosconfigmanagement.googleapis.com"
+}
+
+resource "google_project_service" "csr" {
+  project = var.project_id
+  service = "sourcerepo.googleapis.com"
+}
+
+resource "google_sourcerepo_repository" "acm_repo" {
+  project = var.project_id
+  name    = "eab-acm"
+
+  depends_on = [
+    google_project_service.csr
+  ]
+}
+
+resource "google_service_account" "root-reconciler" {
+  project      = var.project_id
+  account_id   = "root-reconciler"
+  display_name = "root-reconciler"
+}
+
+resource "google_project_iam_member" "root-reconciler" {
+  for_each = toset(["roles/source.reader"])
+
+  project = var.project_id
+  role    = each.value
+  member  = "serviceAccount:${google_service_account.root-reconciler.email}"
+}
+
+resource "google_service_account_iam_binding" "workload-identity" {
+  service_account_id = google_service_account.root-reconciler.name
+  role               = "roles/iam.workloadIdentityUser"
+
+  members = [
+    "serviceAccount:${var.project_id}.svc.id.goog[config-management-system/root-reconciler]",
+  ]
+}
+
+resource "google_gke_hub_feature" "acm_feature" {
+  name     = "configmanagement"
+  project  = var.project_id
+  location = "global"
+
+  depends_on = [
+    google_project_service.acm
+  ]
+}
+
+resource "google_gke_hub_feature_membership" "acm_feature_member" {
+  for_each = toset(var.cluster_membership_ids)
+
+  project    = var.project_id
+  location   = "global"
+  feature    = "configmanagement"
+  membership = regex(local.membership_re, each.key)[2]
+  configmanagement {
+    version = "1.15.0"
```

nit: do we want the current release?

```suggestion
    version = "1.17.2"
```

---

done